.circleci/config.yml

```diff
@@ -34,24 +34,33 @@ commands:
           - v1-deps-{{ checksum "build.sbt" }}
 
       - run:
-          name: Compile
-          command: sbt test:compile
+          name: Compile 2.11
+          command: sbt ++2.11.12 test:compile
+
+      - run:
+          name: Compile 2.12
+          command: sbt ++2.12.11 test:compile
 
       - save_cache:
           paths:
           - ~/.ivy2
           key: v1-deps-{{ checksum "build.sbt" }}
 
       - run:
-          name: Run Tests
-          command: sbt coverage test coverageReport
+          name: Test 2.11
+          command: sbt ++2.11.12 coverage test coverageReport
+
+      - run:
+          name: Test 2.12
+          command: sbt ++2.12.11 coverage test coverageReport
 
       - run:
           when: always
```

Comment:
It looks like no reports are created if the `test` task fails. That is, from previous step:

```sh
sbt coverage test coverageReport
```

If `test` fails, `coverageReport` is not executed, thus no reports are created.

@evbruno, should we remove the `always` setting here then?

Reply:
that makes sense as well